docs/core/testing/unit-testing-with-mstest.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Use MSTest with .NET Core
+description: How to use MSTest with .NET Core
+keywords: MSTest, .NET, .NET Core
+author: ncarandini
+ms.author: wiwagn
+ms.date: 02/09/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: ed447641-3e85-4e50-b7ed-004630048a3e
+---
+
+# Unit testing with MSTest and .NET Core
```

Comment:
On the title: I tried alternatives to 'unit  testing' and anything I tried seemed very contrived. I'd rather leave it as is.

On the TOC: I tried the review site this morning, and I'm not seeing any issues. Can you check again after this update builds?